### DISCUSSION TITLE
Control caligraphy options with Shortcut Composer

### DISCUSSION THREAD

**Discussion by Patpc2** [wojtryb/Shortcut-Composer#95] [Ideas]
Hello,
I really appreciate this plugin. I am a calligrapher. Would it be possible to use the tool angle palette with Krita's calligraphy tools? That would be excellent, because in calligraphy the writing angle is essential.
thank you

**Comment by wojtryb** [comment]
Hi,

this property is not exposed to python, so it cannot be modified from a plugin.
You can ask krita developers to add this support, but I would not hope for too much as this use-case is rather niche.

**Comment by Patpc2** [comment]
Thanks you
 
Envoyé depuis l'application Mail Orange
----------------De : Wojciech Trybus ***@***.***>Envoyé : samedi 02 novembre 2024 11:40À : wojtryb/Shortcut-Composer ***@***.***>Cc : Patpc2 ***@***.***>; Author ***@***.***>Objet : Re: [wojtryb/Shortcut-Composer] short composer (Discussion #95)
Hi,
this property is not exposed to python, so it cannot be modified from a plugin.
You can ask krita developers to add this support, but I would not hope for too much as this use-case is rather niche.
